packages/http-client-java/generator/pom.xml

```diff
@@ -110,7 +110,6 @@
                 <include>src/test/java/**/*.java</include>
               </includes>
 
-              <removeUnusedImports/>
```

Comment:
We can keep this as this is for the generator code itself and not used for generating libraries.